GpsDataCapturer/wear/src/main/res/values/strings.xml

```diff
@@ -8,7 +8,13 @@
     <string name="locationmanager_api" translatable="false">LocationManager API</string>
     <string name="fusedlocationprovider_api" translatable="false">FusedLocationProvider API</string>
     <string name="gps_data" translatable="false">GPS Data</string>
-    <string name="waiting_gps_data" translatable="false">Waiting for GPS Data...</string>
-    <string name="gps_status_not_available">GPS Status: Not Available</string>
-    <string name="satellites_not_available">Satellites: Not Available</string>
+    <string name="waiting_gps_data" translatable="false">Waiting for GPS Data...\n</string>
+    <string name="gps_status_not_available">Not Available</string>
+    <string name="satellites_not_available">Not Available</string>
+    <string name="satellites">Satellites:  </string>
+    <string name="gps_status">GPS Status:  </string>
+    <string name="lat">LAT:  </string>
+    <string name="loc_data">00.000000</string>
```

Comment:
can we rename it to default_location_data?

Reply:
Done.